### ISSUE TITLE
about create model

### ISSUE THREAD

**Issue by GabbyMrH** [mtolhuys/laravel-schematics#39]
An error occurred when I created the model
```
"message": "Use of undefined constant STDIN - assumed 'STDIN' (this will throw an Error in a future version of PHP)",
    "exception": "ErrorException",
    "file": "/Users/gabbymrh/Desktop/my_project/FunnyForum/ForumAPI/vendor/symfony/console/Helper/QuestionHelper.php",
    "line": 118,
```

**Comment by mtolhuys** [comment]
Hi @GabbyMrH, this seems to have been fixed by @rpouls in https://github.com/mtolhuys/laravel-schematics/pull/38

Can you update to 0.10.3 and try again?

**Comment by GabbyMrH** [comment]
thank you,it's woking now.

**Comment by GabbyMrH** [comment]
thank you ,it's working now.

**Comment by GabbyMrH** [comment]
> Hi @GabbyMrH, this seems to have been fixed by @rpouls in #38
> 
> Can you update to 0.10.3 and try again?

thank you ,it's working now.
